### DISCUSSION TITLE
Is there a Porous Flow Master Action System

### DISCUSSION THREAD

**Discussion by jmeier** [idaholab/moose#27499] [Q&A Modules: Porous Flow] [answered]
Dear Community,

Is there something like the "Tensor Mechanics Master Action System" ([link](https://mooseframework.inl.gov/syntax/Modules/TensorMechanics/Master/)) for porous flow? 

Thanks for your help.

**Comment by GiudGiud** [comment]
hello

I think there's this
https://mooseframework.inl.gov/source/actions/PoroMechanicsAction.html

Guillaume

**Reply by jmeier** [reply]
Dear @GiudGiud,

Thanks for the quick answer. Unfortunately, the docu on PoroMechanicsAction is still under construction and I fail to find an example where this action is used within [Physics]. Preferably in combination with [SolidMechanics/QuasiStatic]. I only find a "PoroMechanics"-Action used inside a kernel [here](https://github.com/idaholab/moose/blob/4764993ee032e0dd1efa102bda0e4a1a3d80596d/modules/solid_mechanics/test/tests/poro/vol_expansion_action.i#L74).
Do you know if there is such an example available?

```
[Physics]
  [SolidMechanics/QuasiStatic]
    [./all]
      add_variables = true
      ...
    []
  []
  [PoroMechanics]
      ?????
  []
[]
```

**Reply by GiudGiud** [reply, marked as answer]
The [Physics] syntax is currently being rolled out. 
We have not gotten to PoroMechanics. In fact, Poromechanics being nested under Kernels rather than Modules/PorousFlow makes me think it hasn't been transitioned to the previously preferred syntax either. So you wont find an example with the [Physics] syntax for a while
